sdk/spring/azure-spring-boot-starter-servicebus-jms/README.md

```diff
@@ -30,14 +30,22 @@ Support for AMQP 1.0 in Azure Service Bus means that you can use the queuing and
 ### Configure the app for your service bus
 In this section, you see how to configure your app to use either a Service Bus queue or topic.
 
+#### configuration options
+
+Name | Description 
+|:---|:---
+spring.jms.servicebus.connection-string | Specify the connection string you obtained in your Service Bus namespace from the Azure portal.
+spring.jms.servicebus.topic-client-id | Specify the JMS client ID, which is your Service Bus Subscription ID in the Azure portal.
```

Comment:
I think the `topic-client-id` here means: the value in specified will be diplayed as the SubscriptionName in the Azure portal.
the value is custom self defined,right?